src/Http/Controllers/CP/Collections/CollectionsController.php

```diff
@@ -445,6 +448,19 @@ protected function editFormBlueprint($collection)
             ],
         ];
 
+        if (Statamic::pro() && config('statamic.revisions.enabled')) {
+            $fields['revisions'] = [
+                'display' => __('Revisions'),
+                'fields' => [
+                    'revisions' => [
+                        'type' => 'toggle',
+                        'display' => __('Enable Revisions'),
+                        'instructions' => __('statamic::messages.collection_configure_propagate_instructions'),
```

Comment:
Key needs to be updated (and translation added)

Reply:
I knew I forgot something. Thanks!